blog-posts/gather-metaverse-bump-each-other-2021.md

```diff
@@ -0,0 +1,130 @@
+---
+path: /blog/gather-metaverse-bump-each-other-2021/
+date: 2021-12-21
+title: How we joined the Metaverse with Gather 
+featuredImage: images/gather/teaser.png
+seoMetaText: A few weeks ago, we introduced Gather at Satellytes to fight Zoom Fatigue. Soon after, we saw the magic of the Metaverse happening.
+teaserText: A few weeks ago, we introduced Gather at Satellytes to make our daily remote life more pleasant and to fight Zoom Fatigue. Soon after, we saw the magic of the Metaverse happening.
+attribution:
+    creator: Made with Gather Tiles
+    source: https://github.com/gathertown/mapmaking
+author: Georgios Kaleadis
+authorSummary: CTO at Satellytes
+---
+
+## Introduction
+A few weeks ago, we introduced [Gather](https://www.gather.town/) at Satellytes to make our daily remote life more pleasant and to fight [Zoom Fatigue](https://en.wikipedia.org/wiki/Zoom_fatigue). Gather describes itself as a **_"platform that combines a 2D game-like interface with video-chat"_**. 
+
+![Official gather screenshot with people hanging around](images/gather/gather-example-official.png)
```

Comment:
I think I would push this back a little bit. I personally found this hindering me getting into the article.

Reply:
Thanks, I moved it down to the entire introduction. I still want to give a first image to the reader's mind.

![Screen Shot 2021-12-22 at 15 47 13](https://user-images.githubusercontent.com/1701755/147110523-df17ed03-cf41-4f92-9f11-4767c7d45e97.png)